.buildkite/pipeline.yml

```diff
@@ -787,6 +787,29 @@ steps:
         agents:
           slurm_mem: 20GB
 
+  - group: "CUDA"
+    steps:
+
+      - label: "CUDA: CUDA implicit baro wave"
+        key: "cuda_implicit_barowave_ref"
+        command:
+          - "julia --project -e 'using CUDA; CUDA.versioninfo()'"
+          - "julia --color=yes --project=examples examples/hybrid/driver.jl --job_id cuda_implicit_barowave_ref --dt 580secs --t_end 10days --dt_save_to_disk 2days --initial_condition DryBaroclinicWave --dt_save_to_sol Inf"
+        artifact_paths: "cuda_implicit_barowave_ref/*"
+        agents:
+          slurm_gpus: 1
+
+      - label: "CUDA: CPU implicit baro wave"
+        key: "cpu_implicit_barowave_ref"
+        command: "julia --color=yes --project=examples examples/hybrid/driver.jl --job_id cpu_implicit_barowave_ref --dt 580secs --t_end 10days --dt_save_to_disk 2days --initial_condition DryBaroclinicWave --dt_save_to_sol Inf"
```

Comment:
Why do we need to add a new CPU baroclinic wave as a reference? Can't we use one of the baroclinic wave integration tests we already have in CI?

Reply:
I had thought about that, but these jobs are primarily for comparing CPU/GPU implementations, I figured it may be easier to just leave it decoupled from the other jobs

Reply:
Any suggestions?

Reply:
What are the main differences in the setups with the proposed job and one of the existing ones? Is it `t_end` mainly or something else?

Reply:
I don't think there are differences, actually, but I think the purpose of the other job is for visualizing the output, and maybe doing science?

Reply:
I see. I would not be too excited to add another ad-hoc CPU test that it does not test any different functionality, but it's only for the purpose of comparing results with the GPU test. We often find ourselves in the position to reduce CI jobs. If it is ok with @szy21 , it's fine by me.

Reply:
Can we just use the results from the existing job?

Reply:
Sure, I can delete the duplicate